native/src/main/java/tech/sourced/babelfish/DriverResponse.java

```diff
@@ -0,0 +1,84 @@
+package tech.sourced.babelfish;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.eclipse.jdt.core.dom.CompilationUnit;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+
+/**
+ * Class for the java driver response
+ */
+public class DriverResponse {
+    final public String driver;
+    final public String language;
+    final public String languageVersion;
+    public String status = "ok";
+    public ArrayList<String> errors = new ArrayList<String>(0);
+    @JsonProperty("AST")
```

Comment:
the AST property is `"ast"` in the json. Does this lowercase the name? Otherwise, it should be changed to lowercase